contract/src/main/kotlin/io/provenance/scope/loan/contracts/RecordLoanContract.kt

```diff
@@ -120,14 +125,6 @@ open class RecordLoanContract(
 
     @Function(invokedBy = PartyType.OWNER)
     @Record(LoanScopeFacts.eNote)
-    open fun recordENote(@Input(LoanScopeFacts.eNote) eNote: ENote) = eNote.also {
-        validateRequirements(VALID_INPUT) {
-            if (existingENote.isSet()) {
-                requireThat((existingENote.eNote.checksum == it.eNote.checksum) orError
-                    "ENote with a different checksum already exists on chain for the specified scope; ENote modifications are not allowed!"
-                )
-            }
-            eNoteValidation(eNote)
-        }
-    }
+    @SkipIfRecordExists(LoanScopeFacts.eNote)
```

Comment:
@dkneisly-figure can you confirm that we want this annotation for `RecordLoanContract.recordENote`?

Reply:
Let's do it this way to start